test/native/bs-css/dune

```diff
@@ -0,0 +1,3 @@
+(library
+ (name StyledPpxTestNativeBSCSS)
+ (modules_without_implementation js css_AtomicTypes css_Core))
```

Comment:
This line is the trick, with it we can typecheck but not build, which is exactly what we want